source/lib/ramsaySt.py

```diff
@@ -55,13 +55,13 @@ def setGlyph(self, glyph):
                 baseName = RamsayStData.getBaseGlyph(glyph.name)
                 leftGlyphName, rightGlyphName = RamsayStData.get(baseName, ("n", "n"))
 
-            if leftGlyphName in layer:
+            if leftGlyphName in layer.keys():
```

Comment:
why? this is a fontParts layer and it has contains already, no need to convert it first to a set of keys 
(same in in line 64)

Reply:
because of this
```
Observer '<__main__.RamsaySts object at 0x51c56d910>' with '_glyphEditorDidSetGlyphInternalCallback' failed: Glyph names must be at least one character long.
```

Reply:
resolved in 9b6ab45a48768c2070f483d45b837f951c104d10, as discussed